docs/app/pages/components/components-sections/timeline/timeline/timeline.component.html

```diff
@@ -0,0 +1,31 @@
+<hr>
+
+<p>
+    The Timeline component will display a vertical list of events. It will be defined by a <code>timeline</code> tag. Each event will be defined by a <code>timeline-event</code> tag.
+</p>
+
+<p>
+    For each event a badge and panel will be displayed. Information to be displayed in the badge and panel will be supplied as inputs to the Timeline event component.
+</p>
+
+<p>The following attributes can be used to customise each event:</p>
```

Comment:
"The following attributes can be used to configure the `ux-timeline-event` components:"

Also note that the content of the component